repos/system_upgrade/common/actors/systemd/setsystemdservicesstates/libraries/setsystemdservicesstate.py

```diff
@@ -0,0 +1,31 @@
+from leapp.libraries.stdlib import api, CalledProcessError, run
+from leapp.models import SystemdServicesTasks
+
+
+def _try_set_service_state(command, unit):
```

Comment:
@matejmatuska  when refered previousl to the `unit` vs `service` , I had in my mind the argument as well

Reply:
Of course, I didn't notice it.